samples/daily-prebuilt/permissions-can-admin/package-lock.json

```diff
@@ -13,19 +13,6 @@
         "next": "13.4.16",
         "react": "18.2.0",
         "react-dom": "18.2.0"
-      },
```

Comment:
I am not actually sure why all of this was removed... but I _did_ do a clean install locally - maybe some reps could have just been purged? 🤔